src/routes/shelters.js

```diff
@@ -302,6 +302,53 @@ router.get('/:id/delete_review', async(req, res) => {
     return;
 });
 
+router.get('/:id/edit_review', async (req, res) => {
+  let shelterId = req.params.id;
+  let loggedIn = helpers.isAuthenticated(req);
```

Comment:
Also, if the use is not logged in, then I think it would be better to redirect them to either the home page or back to the shelter page. What do you think?